src/cmd/manage_nsfs.js

```diff
@@ -14,19 +14,26 @@ const SensitiveString = require('../util/sensitive_string');
 const ManageCLIError = require('../manage_nsfs/manage_nsfs_cli_errors').ManageCLIError;
 const ManageCLIResponse = require('../manage_nsfs/manage_nsfs_cli_responses').ManageCLIResponse;
 const bucket_policy_utils = require('../endpoint/s3/s3_bucket_policy_utils');
+const { SimpleWAL, NewlineReader } = require('../util/wal');
+const eeadm = require('../util/tapecloud_utils');
+const { parse_decimal_int } = require('../endpoint/s3/s3_utils');
+const os = require('os');
 
 const TYPES = {
     ACCOUNT: 'account',
     BUCKET: 'bucket',
-    IPWHITELIST: 'whitelist'
+    IPWHITELIST: 'whitelist',
+    TAPE: 'tape',
```

Comment:
Perhaps better refer to this as GLACIER instead of tape?
This can go to any media that supports migrate/restore commands...